src/stringify.ts

```diff
@@ -13,7 +14,12 @@ const defaultOpts: Partial<RemarkStringifyOptions> = {
 const defaultProcessor = unified()
   .use<RemarkStringifyOptions[]>(remarkStringify)
   .use(jiraBlocks)
-  .use(frontmatter, ['yaml']);
+  .use(frontmatter, ['yaml'])
+  .use(resolver, {
+    async resolver() {
+      return {};
+    },
```

Comment:
do we actually need to add a no-op resolver?

Reply:
That's how the API looks like, but well, I think I could indeed make it optional.